cmd/main.go

```diff
@@ -268,5 +269,7 @@ func main() {
 		}
 	}()
 	<-c
+	logrus.Info("Doing seeded content cleanup...")
+	lProv.CleanupSeeded(seededUUIDs)
```

Comment:
Do we actually need this cleanup?

Reply:
The end note was to keep it for now.
@leecalcote  can you confirm?

Reply:
The advantage to have this is:
If the volume attached to meshery server is persisted across meshery restarts in those case we will end up with duplicate in the database.